code/modules/organs/organ_external.dm

```diff
@@ -450,7 +450,7 @@
 
 	//Bone fracurtes
 	var/datum/species/species = src.species || owner.species
-	if(config.bones_can_break && brute_dam > min_broken_damage * config.organ_health_multiplier && is_organic() && !(species.anatomy_flags & NO_BONES))
+	if(config.bones_can_break && brute_dam > min_broken_damage * config.organ_health_multiplier * (has_mutation(M_EXOSKELETON) ? 0.5 : 1) && is_organic() && !(species.anatomy_flags & NO_BONES))
```

Comment:
Won't this have very little/no effect on bones breaking due to the 0.5 brute resistance? 
If brute damage was 10 and min_broken_damage is 15, then organ_health_multipler is 1 for simplicity:

10 * 0.5 < 15 * 1 * 0.5
versus
10 < 15 * 1 * 1
Neither would break. Basically the exoskeleton is halving the damage threshold required to break the bone, but you're also halving the damage so it's a non-factor.

Reply:
You're right here, I think I'll reduce the brute resist somewhat so it *is* easier to get a break, in addition to the organ damage.